### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sergeysova**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+## Reporting Issues and Asking Questions
+Before opening an issue, please search the [issue tracker](https://github.com/zerobias/effector/issues) to make sure your issue hasn't already been reported.
+
+## Development
+
+Visit the [Issue tracker](https://github.com/zerobias/effector/issues) to find a list of open issues that need attention.
+
+Fork, then clone the repo:
+```
+git clone https://github.com/your-username/effector.git
```
Comment (zerobias) [inline CONTRIBUTING.md]: [gitpod](https://gitpod.io/#https://github.com/zerobias/effector) is [configured](https://github.com/zerobias/effector/blob/master/.gitpod.yml) as well

**Comment by doasync** [comment]
There are some grammar mistakes

**Comment by sergeysova** [comment]
@doasync can you fix it?

**Comment by doasync** [comment]
> @doasync can you fix it?
Yep

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+### Tests
+
+Just use `yarn test` and `yarn test:watch`.
+
+### New Features
+
+Please open an issue with a proposal for a new feature or refactoring before starting on the work.
+We don't want you to waste your efforts on a pull request that we won't want to accept.
+
+Before opengin an feature request, please read the [exists RFC](https://github.com/zerobias/effector/tree/master/rfc) to make sure your feature hasn't already requested.
```
Comment (sanohin) [inline CONTRIBUTING.md]: opening a

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
+# Contributing
+We are open to, and grateful for, any contributions made by the community.  By contributing to Effector, you agree to abide by the [code of conduct](https://github.com/zerobias/effector/blob/master/CODE_OF_CONDUCT.md).
```
Comment (sanohin) [inline CONTRIBUTING.md]: We are open to and

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+Please open an issue with a proposal for a new feature or refactoring before starting on the work.
+We don't want you to waste your efforts on a pull request that we won't want to accept.
+
+Before opengin an feature request, please read the [exists RFC](https://github.com/zerobias/effector/tree/master/rfc) to make sure your feature hasn't already requested.
+
+## Submitting Changes
+
+* Open a new issue in the [Issue tracker](https://github.com/zerobias/effector/issues).
+* Fork the repo.
+* Create a new feature branch based off the `master` branch.
```
Comment (sanohin) [inline CONTRIBUTING.md]: based on

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -17,6 +17,25 @@ Another way is to use [gitpod](https://gitpod.io):
...
+- `docs` is a directory with .md files to build [effector.now.sh](https://effector.now.sh)
+- `src` contains all source files of effector, effector-react, ...
+- `packages` contains all files required to publish npm-packages
+- `tasks` is a directory with scripts to build npm-packages
+- `website` contains source files of [effector.now.sh](https://effector.now.sh)
+
+
+### `yarn build`
+
+This command uses [`./tasks`](https://github.com/zerobias/effector/tree/master/tasks) to generate package.json for each npm-package from `./packages`.
```
Comment (zerobias) [inline CONTRIBUTING.md]: tasks more like "everything you need to do to publish a package" including compilation of sources and copying license files

**Pull request merged**